algod/build/config.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
This file might be optional, I think it is created during the install process. That would be better since it would have new fields.